packages/flutter_plugin_android_lifecycle/README.md

```diff
@@ -13,10 +13,6 @@ major version of the Android `Lifecycle` API they expect.
 |-------------|---------|
 | **Support** | SDK 16+ |
 
-## Installation
-
-Add `flutter_plugin_android_lifecycle` as a [dependency in your pubspec.yaml file](https://flutter.dev/using-packages/).
-
```

Comment:
There's several of these where we're removing the `using-packages` link, is this intentional?

Reply:
I dropped this in response to @stuartmorgan's mention in https://github.com/flutter/packages/pull/6963#discussion_r1648273732 that this section isn't super useful with pub.dev surfacing installation information. Perhaps I misunderstood how much I should remove though.

Reply:
These changes are what I had in mind, thanks!